### DISCUSSION TITLE
Nice mod

### DISCUSSION THREAD

**Discussion by littleduckcoding** [Woodi-dev/Among-Us-Sheriff-Mod#89] [General]
Thanks for your mod. It's really nice and fun to play.

I am curious about creating this mod, for example, in Player.cs file, how can I unscramble "FFGALNAPKCD" type? Are there any references to this and many other similar things in these different files?

**Comment by Woodi-dev** [comment]
FFGA... is the obfuscated name for the PlayerControl class in the game
